src/renderer/components/LoginForm.tsx

```diff
@@ -398,17 +423,24 @@ export function ConfigureProgressDialog({
     () => {
       ;(async () => {
         try {
-          if (window.__selectedAccountId === undefined) {
-            throw new Error('No account selected')
-          }
-
-          await BackendRemote.rpc.batchSetConfig(window.__selectedAccountId, {
+          // Prepare initial configuration
+          const initialConfig: { [key: string]: string } = {
             ...credentials,
             verified_one_on_one_chats: '1',
-          })
-          await BackendRemote.rpc.configure(window.__selectedAccountId)
+          }
+
+          // Select "Device Messages" chat as the initial one. This will serve
+          // as a first introduction to the app after they've entered
+          const deviceChatId = await getDeviceChatId(accountId)
+          if (deviceChatId) {
+            initialConfig['ui.lastchatid'] = String(deviceChatId)
+          }
```

Comment:
this needs to be done after configuration, also might be better to directly user the settings manager as the value is also fetched from there then we don't need to care for updating the cached value in the settings manager.

Reply:
> this needs to be done after configuration (before that device chat does not exist, just tested it)

Ah, thanks for testing! In my case it actually worked, must be a race condition? I'll move it further to avoid it.

Reply:
imu, [`configure()` waits for device chat being created](https://github.com/deltachat/deltachat-core-rust/blob/main/src/configure.rs#L486) [by another thread](https://github.com/deltachat/deltachat-core-rust/blob/main/src/configure.rs#L182), so after `configure()` has finished we should be safe.

but while `configure()` runs we may or may not have the chat

Reply:
(this is about the bare core calls, jsonrpc may add further pitfalls :)

in the case of configure it only calls stop_io and start_io automatically.